index.js

```diff
@@ -250,12 +272,16 @@ DB.prototype.get = function (key, opts, cb) {
   }
   this.kv.get(key, opts, function (err, docs) {
     if (err) return cb(err)
-    // Filter deletions & map to old expected format.
-    docs = filterObj(docs, function (key, value) {
-      return !value.deleted
-    })
-    docs = mapObj(docs, function (key, value) {
-      return value.value
+    docs = mapObj(docs, function (version, value) {
+      if (value.deleted) {
+        return {
+          id: key,
+          version: version,
+          deleted: true
+        }
+      } else {
+        return value.value
+      }
```

Comment:
Let's keep this consistent:

if (doc.d) {
  return xtend(doc.v, {deleted: true})
} else {
  return doc.v
}